### PR TITLE
Add scene for accept friendship

### PR DESCRIPTION
**Pull request by su-chang**

### CODE REVIEW

File: src/schemas/friendship.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export type FriendshipPayloadConfirm = FriendshipPayloadBase & {
 
 /** @hidden */
 export type FriendshipPayloadReceive = FriendshipPayloadBase & {
+  scene?    : string,
```
Comment (windmemory) [inline src/schemas/friendship.ts]: Can we make this as an enum?

Comment (su-chang) [reply src/schemas/friendship.ts]: OK

**Comment by windmemory** [comment]
Please fix the build.

**Comment by su-chang** [comment]
@huan this PR will make padplus accept friend request by different scenes.

**Comment by huan** [comment]
I added a `Card = 17, // 名片分享` and another `QQ`, with a type rename.

Will merge it first and then continue refining it.

**Pull request merged**